app/services/health_check_service.rb

```diff
@@ -1,6 +1,6 @@
 class HealthCheckService
-  def self.call
-    database_healthy? && short_term_persistence_healthy? && cfe_healthy?
+  def self.call(check_cfe:)
+    database_healthy? && short_term_persistence_healthy? && (!check_cfe || cfe_healthy?)
```

Comment:
This expression is awfully clever and I had to have a good sip of coffee and a proper think to reason the logic through... It scares me.

Reply:
Ok, I've split it up into two more digestible expressions.